Issue building EssX because of Spigot API update: getRecipe()

### Help request

**Problem**
<!-- What problem did you encounter? -->

Build with Maven spits the error:

[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/runner/work/Essentials/Essentials/Essentials/test/com/earth2me/essentials/FakeServer.java:[50,8] com.earth2me.essentials.FakeServer is not abstract and does not override abstract method getRecipe(org.bukkit.NamespacedKey) in org.bukkit.Server

--- a/Essentials/test/com/earth2me/essentials/FakeServer.java
+++ b/Essentials/test/com/earth2me/essentials/FakeServer.java
@@ -914,6 +914,11 @@ public class FakeServer implements Server {
     }
 
     @Override
+    public Recipe getRecipe(NamespacedKey namespacedKey) {
+        throw new UnsupportedOperationException("Not supported yet.");
+    }
+
+    @Override
     public Iterator<Recipe> recipeIterator() {
         throw new UnsupportedOperationException("Not supported yet.");
     }

